mailpoet/lib/EmailEditor/Integrations/MailPoet/PersonalizationTagManager.php

```
@@ -16,22 +20,96 @@ class PersonalizationTagManager {
+
+    $this->wp->applyFilters('mailpoet_automation_email_extend_personalization_tags', $registry, $availableSubjects);
+  }
+
+  /**
+   * Extend tags for REST API requests to personalization tags endpoint.
+   * This is necessary because the registry is initialized early and preloading fetches from it immediately.
+   *
+   * @return void
+   */
+  public function maybeExtendTagsForRestRequest(): void {
+    $this->wp->addFilter('rest_pre_dispatch', function($result, $server, $request) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$server'. (undefined)

(UnusedFormalParameter)